src/features/credits/components/CreditLog.tsx

```
@@ -225,8 +225,17 @@ function getEntryTitle(entry: CreditEntry): string {
         ? 'Applied for Project'
         : 'Submitted to a Bounty';
     case 'MONTHLY_CREDIT':
-      return 'Credits Renewed';
+      const now = new Date();
+      const effectiveDate = new Date(entry.effectiveMonth);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)